docs/basics.md

```diff
@@ -238,7 +238,7 @@ Here is an example of backends and servers definition:
 
 # Launch
```

Comment:
So `# Launch` becomes `# Configuration`, right ?
But not sure to understand about add a section... 
Do you wa